config/examples/Artillery/Sidewinder X2/Configuration.h

```diff
@@ -2588,6 +2588,11 @@
   #define USE_MKS_GREEN_UI
 #endif
 
+//
+// CR-6 OEM touch screen. A DWIN display with touch.
+//
+//#define DWIN_CREALITY_TOUCHLCD
+
```

Comment:
Should these also be in the main Marlin repo's config? I didn't catch these because they are missing.

Reply:
Submissions here are rebased against `import-2.0.x`.

Reply:
I meant `bugfix-2.0.x` in the main Marlin repo.